server/src/main/java/org/opensearch/search/profile/query/CollectorResult.java

```diff
@@ -113,6 +164,19 @@ public CollectorResult(StreamInput in) throws IOException {
             CollectorResult child = new CollectorResult(in);
             this.children.add(child);
         }
+        if (in.getVersion().onOrAfter(Version.V_3_0_0)) {
```

Comment:
```suggestion
        if (in.getVersion().onOrAfter(Version.V_2_9_0)) {
```

Reply:
Thanks @reta for the change request. Our target for the general availability of concurrent search is version `2.10`. We may need to wait until the `2.10` branch and version name are created.

Reply:
Thanks for the clarification, @reta.